src/ResourceManager/IotHub/Commands.IotHub/help/New-AzureRmIotHubExportDevices.md

```diff
@@ -27,8 +27,8 @@ https://azure.microsoft.com/en-us/documentation/articles/iot-hub-bulk-identity-m
 ## EXAMPLES
 
 ### Example 1 Issue an export device request.
-```
-PS C:\> New-AzureRmIotHubExportDevices -ResourceGroupName "myresourcegroup" -Name "myiothub" -ExportBlobContainerUri "https://mystorageaccount.blob.core.windows.net/?sv=2015-04-05&ss=bfqt&sr=c&srt=sco&sp=rwdl&se=2016-10-27T04:01:48Z&st=2016-10-26T20:01:48Z&spr=https&sig=QqpIhHsIMF8hNuFO%3D" -ExcludeKeys $true
+```azurepowershell
+PS C:\> New-AzureRmIotHubExportDevices -ResourceGroupName "myresourcegroup" -Name "myiothub" -ExportBlobContainerUri "https://mystorageaccount.blob.core.windows.net/?sv=2015-04-05&ss=bfqt&sr=c&srt=sco&sp=rwdl&se=2016-10-27T04:01:48Z&st=2016-10-26T20:01:48Z&spr=https&sig=QqpIhHsIMF8hNuFO%3D" -ExcludeKeys:$True
```

Comment:
Good catch.  Since this is a switch parameter it is better canonical usage to simply include the parameter, rather than explicitly set it to true, i.e.:

```powershell
... -ExcludeKeys
```

rather than

```powershell
... -ExcludeKeys:$true
```

Reply:
For the sake of the example, can we explicitly set it to false? Otherwise it seems difficult to know how to do that.

Reply:
In this case, since this is a New-* cmdlet, the way to set it to false is to not include the switch